### PR TITLE
dev-python/dkimpy: version bump 1.0.4

### PR DESCRIPTION
**Pull request by janko-j1**
This version bump introduces new optional dependencies

### CODE REVIEW

**Comment by thesamesam** [comment]
You need the full URL in Bug tags 👍

**Comment by janko-j1** [comment]
I needed to put it in the commit message ^^

**Comment by janko-j1** [comment]
Thanks to Alessandro-Barbieri for pointing me to https://projects.gentoo.org/qa/policy-guide/dependencies.html#pg0001

**Comment by janko-j1** [comment]
I re-added ~arm64 as bug https://bugs.gentoo.org/723444 is resolved.

File: dev-python/dkimpy/dkimpy-1.0.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
```
Comment (mgorny) [inline dev-python/dkimpy/dkimpy-1.0.4.ebuild:6]: Why not 3.9? ;-)

File: dev-python/dkimpy/dkimpy-1.0.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE=""
+
+DEPEND="
+	dev-python/dnspython[${PYTHON_USEDEP}]
+"
+RDEPEND="${DEPEND}"
```
Comment (mgorny) [inline dev-python/dkimpy/dkimpy-1.0.4.ebuild:23]: Do you really need it outside `test?`, and then `disutils_enable_tests` takes care of adding it.

File: dev-python/dkimpy/dkimpy-1.0.4.ebuild [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+distutils_enable_tests unittest
+
+DEPEND+="
+	test? (
+	dev-python/authres[${PYTHON_USEDEP}]
```
Comment (mgorny) [inline dev-python/dkimpy/dkimpy-1.0.4.ebuild:29]: Indent.

File: dev-python/dkimpy/dkimpy-1.0.4.ebuild (hunk 1 above)
Comment (janko-j1) [reply dev-python/dkimpy/dkimpy-1.0.4.ebuild:6]: dev-python/pynacl and dev-python/authres do not have python3_9. But is this a good reason ?

File: dev-python/dkimpy/dkimpy-1.0.4.ebuild (hunk 2 above)
Comment (janko-j1) [reply dev-python/dkimpy/dkimpy-1.0.4.ebuild:23]: It is listed in install_requires so I figured it must be there ?

Comment (janko-j1) [reply dev-python/dkimpy/dkimpy-1.0.4.ebuild]: It is actually needed in RDEPEND

File: dev-python/dkimpy/dkimpy-1.0.4.ebuild (hunk 1 above)
Comment (janko-j1) [reply dev-python/dkimpy/dkimpy-1.0.4.ebuild:6]: dev-python/pynacl and dev-python/authres (test dependencies) do not have python3_9.

Comment (mgorny) [reply dev-python/dkimpy/dkimpy-1.0.4.ebuild:6]: Both done now.

File: dev-python/dkimpy/dkimpy-1.0.4.ebuild [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE=""
+
+RDEPEND="
+	dev-python/dnspython[${PYTHON_USEDEP}]
+"
+
+distutils_enable_tests unittest
+
+DEPEND+="
```
Comment (mgorny) [inline dev-python/dkimpy/dkimpy-1.0.4.ebuild:26]: BDEPEND

Comment (janko-j1) [reply dev-python/dkimpy/dkimpy-1.0.4.ebuild]: Right, I overlooked https://dev.gentoo.org/~mgorny/python-guide/distutils.html#dependencies.

**Pull request closed without merging**